runtime/compiler/build/rules/win-msvc/filetypes.mk

```diff
@@ -89,6 +89,22 @@ endef # DEF_RULE.pasm
 
 RULE.pasm=$(eval $(DEF_RULE.pasm))
 
+#
+# Compile .nasm file into .obj file
+#
+define DEF_RULE.nasm
+$(1): $(2) | jit_createdirs
+	$(NASM_CMD) $(NASM_OBJ_FORMAT) $$(patsubst %,-D%=1,$$(NASM_DEFINES)) $$(patsubst %,-I'%/',$$(subst \,/,$$(NASM_INCLUDES))) -o $$(subst \,/,$$@) $$(subst \,/,$$<)
```

Comment:
The "/" vs "\\" should be the reason why sanity test failed. Replace it with following line works on my machine.
```$(NASM_CMD) $(NASM_OBJ_FORMAT) $$(patsubst %,-D%=1,$$(NASM_DEFINES)) $$(patsubst %,-I%,$$(call FIXPATH,$$(NASM_INCLUDES))) -o $$(subst \,/,$$@) $$(subst \,/,$$<)```